_docs-sources/intro/tool-fundamentals/docker.md

```diff
@@ -0,0 +1,24 @@
+---
+'title': 'Docker'
+---
+
+# Docker
+
+[Docker](https://www.docker.com) is an open source tool you can use to run _containers_ and define _container images_ as
```

Comment:
This was all copied straight from the [How to use the Gruntwork Infrastructure as Code Library](https://gruntwork.io/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library/#docker). I'm generally in favor of keeping the content as is for MVP and separating out changes like this, but I'm open for discussion and disagreement!

Reply:
That's reasonable. Given that it might require a more thorough survey of the content to catch all instances, let's just keep this in mind as we formalize our style guides for docs content.